From c4d7401059eff12f9fc4d4774d3e27aed50ba624 Mon Sep 17 00:00:00 2001
From: xuhuicong <xhc@rock-chips.com>
Date: Fri, 26 Feb 2016 16:03:00 +0800
Subject: [PATCH] video: rockchip: hdmi: v1: modify double free to avoid
 compile warning

rockchip_hdmiv1.c:375:1-6: WARNING: invalid free of devm_ allocated data

Change-Id: I0ab0b3d8f30a9c106e67f33ae93bc1df1b333fc2
Signed-off-by: xuhuicong <xhc@rock-chips.com>
---
 drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1.c b/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1.c
index bcc01b5f3070..88610b8352d4 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1.c
@@ -372,7 +372,6 @@ static int rockchip_hdmiv1_probe(struct platform_device *pdev)
 failed1:
 	rockchip_hdmi_unregister(hdmi_dev->hdmi);
 failed:
-	kfree(hdmi_dev);
 	hdmi_dev = NULL;
 	dev_err(&pdev->dev, "rk3288 hdmi probe error.\n");
 	return ret;
-- 
2.35.3

